python/paddle/tensor/manipulation.py

```diff
@@ -1447,7 +1447,7 @@ def expand(x, shape, name=None):
     def get_attr_expand_shape(list_expand_shape):
         attrs_expand_shape = []
         for idx, shape in enumerate(list_expand_shape):
-            if isinstance(shape, Variable):
+            if isinstance(shape, Variable) or shape == -1:
```

Comment:
Add comments for what meaning of -2 ?

Reply:
this is add by Lilong.
-2 is only a sign which means dynamic shape.

Reply:
i will add some comment.